module/VuFind/src/VuFind/Controller/AlmaController.php

```diff
@@ -281,19 +284,17 @@ protected function webhookUser($requestBodyJson)
         } elseif ($method == 'DELETE') {
             $user = $this->userTable->getByCatalogId($primaryId);
             if ($user) {
-                $rowsAffected = $user->delete();
-                if ($rowsAffected == 1) {
+                try {
+                    $this->purgeUserData($user);
```

Comment:
I don't think the return value of the old delete method was very meaningful or reliable, so I've opted to replace a return value check with a try..catch. I can't test any of this, but hopefully the changes are sufficiently straightforward!